third_party/terraform/tests/resource_dataproc_cluster_test.go.erb

```diff
@@ -582,6 +582,28 @@ func TestAccDataprocCluster_withNetworkRefs(t *testing.T) {
 	})
 }
 
+<% unless version == 'ga' -%>
+func TestAccDataprocCluster_withEndpointConfig(t *testing.T) {
+	t.Parallel()
+
+	var cluster dataproc.Cluster
+	rnd := acctest.RandString(10)
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		Steps: []resource.TestStep{
+			{
```

Comment:
If you add another step to modify the value from true to false, that should verify whether my comment above actually applies

Reply:
Turns out the field is only available on create, meaning no need for this